### ISSUE TITLE
vite: error when initializing ENSInstance (imported.GraphQLClient is not a constructor)

### ISSUE THREAD

**Issue by appleseed-iii** [ensdomains/ensjs-v3#90]
in a vite application loading ensjs I get the following error when initializing a new `ENSInstance`:

```
TypeError: imported.GraphQLClient is not a constructor
    at GqlManager.setUrl (GqlManager.mjs:68:21)
    at async ENS.setProvider (index.mjs:127:5)
```

I'm running the following package versions:
`"@ensdomains/ensjs": "^3.0.0-alpha.39"`
`"vite": "^3.1.8"`

**Comment by appleseed-iii** [comment]
[here's](https://github.com/appleseed-iii/vite-ensjs-v3) a minimal reproduction. Just run the app with `yarn dev` and then check the console.log

**Comment by appleseed-iii** [comment]
After doing a little digging I found a fix for this. I put the fix here: https://github.com/appleseed-iii/vite-ensjs-v3/pull/1/files

The root problem is that vite doesn't easily bundle CJS packages that are imported via ESM packages. In this case `graphql-request` (CJS) is imported via the ESM ens package.

[Documentation from vite for the fix](https://vitejs.dev/config/dep-optimization-options.html#optimizedeps-exclude)
<img width="721" alt="image" src="https://user-images.githubusercontent.com/80423742/208502806-6560d0f1-0457-43b3-94cd-c2acc63768c3.png">

Closing this since it's not ensjs related
